fineract-provider/src/main/java/org/apache/fineract/organisation/monetary/data/CurrencyData.java

```diff
@@ -97,4 +89,23 @@ private String generateDisplayLabel() {
     public String getName() {
         return name;
     }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (!(o instanceof CurrencyData)) return false;
+        CurrencyData that = (CurrencyData) o;
+        return decimalPlaces == that.decimalPlaces &&
+                code.equals(that.code) &&
+                name.equals(that.name) &&
+                inMultiplesOf.equals(that.inMultiplesOf) &&
```

Comment:
@awasum this is missing `null` checks, and it's also why the build is failing for you on this PR... just use `[Objects.equals()](https://docs.oracle.com/javase/8/docs/api/java/util/Objects.html#equals-java.lang.Object-java.lang.Object-)` instead!

Reply:
Thanks..Will handle this later on Friday.